CalculatorKataTest/CalculatorKataTestShould.cs

```diff
@@ -65,14 +65,27 @@ public void SumNumbersWithNegativeNumbers()
         {
             var obj = new Program();
 
-            //obj.add("1,4,-1");
-
             ArgumentException ex = Assert.Throws<ArgumentException>(() => obj.add("1,-4,-1"));
 
             Assert.Equal("negatives not allowed:-4, -1", ex.Message);
 
         }
 
+        [Fact]
+        public void SumNumbersAndIgnoreBigNumber()
```

Comment:
are there no other cases that you should test? what old code might this "break" that we should also test?